chat/message-feed/src/MessageFeed.stories.tsx

```diff
@@ -189,3 +221,38 @@ export const ChangingMessages: StoryType<typeof MessageFeed> = ({
     </div>
   );
 };
+export const ChangingMessages: StoryObj<MessageFeedStoryProps> = {
+  render: ChangingMessagesComponent,
+};
+
+export const CompactVariant: StoryObj<MessageFeedStoryProps> = {
+  render: Template,
+  args: {
+    children: (
```

Comment:
Curious, do you need to pass children? It looks like the default in `Template` is the same as what you're passing to the children prop here

Reply:
weird, is children required?